src/bindings/js/node/src/addon.cpp

```diff
@@ -15,6 +15,7 @@
 #include "node/include/preprocess/preprocess.hpp"
 #include "node/include/tensor.hpp"
 #include "openvino/openvino.hpp"
+#include "node/include/node_wrap.hpp"
```

Comment:
Please install `clang` and run `make -j44 clang_format_fix_all`.
This line will be probably moved up